### DISCUSSION TITLE
How to enable a "pause" during replay?

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [NickeManarin/ScreenToGif#1301] [Q&A] [answered]
Hi
I made some gifs which I use in my intranet, and it works fine. I also set the option "Don't replay / loop at the end", and it is displayed only once.

Is there an option / a feature where I can enable to "pause" the Gif during display? Every "movie" has this feature - can it be added to a GIF too?

Thanks for an answer - and especially for this great tool!

**Comment by pawlos** [comment, marked as answer]
@Pierre-de-la-Verre If I understand your request correctly, that's usually would be a feature of the player that displays the gif.

**Comment by Pierre-de-la-Verre** [comment]
Thanks, pawlos
to show the right way. A web-search for "firefox gif paus" shows a lot of solutions.
